README.md

```diff
@@ -81,6 +81,7 @@ Repository  |   Description |   License |
 |   [Git and GitHub](https://github.com/tiimgreen/github-cheat-sheet)  |    A list of cool features of Git and GitHub.   |   [MIT](https://github.com/tiimgreen/github-cheat-sheet/blob/master/LICENSE ) |
 |   [Python](https://github.com/gto76/python-cheatsheet)  |    Comprehensive Python Cheatsheet   |   No License  |
 |   [Vim](https://github.com/pawelbr/vim-cheatsheet)  |    Shortcuts that will help you to become an advance VIM user.   |   [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode ) |
+|   [Bootstrap 5 Cheatsheet](https://github.com/themeselection/bootstrap-cheatsheet)  |  An interactive list of Bootstrap 5 classes, variables, and mixins  |  [MIT](https://github.com/tiimgreen/github-cheat-sheet/blob/master/LICENSE ) |
```

Comment:
```suggestion
|   [Bootstrap 5](https://github.com/themeselection/bootstrap-cheatsheet)  |  An interactive list of Bootstrap 5 classes, variables, and mixins  |  [MIT](https://github.com/themeselection/bootstrap-cheatsheet/blob/master/LICENSE.md ) |
```

Reply:
1. Since the entry is in the "CheatSheets" category no need to add  _Cheatsheet_ in the name.  
2. Please arrange the entry in the alphabetical order i.e. after **Async-JavaScript** 
3. Please include the link to the repository's own License file which you wish to add.

Reply:
Ok, I will change it.👍